lib/checks/aria/required-parent.js

```diff
@@ -34,11 +34,15 @@ function getAriaOwners(element) {
 		o = null;
 
 	while (element) {
-		if (element.id) {
-			o = document.querySelector('[aria-owns~=' + axe.commons.utils.escapeSelector(element.id) + ']');
+		if (element.getAttribute('id')) {
+			o = document.querySelector(
+				'[aria-owns~=' + 
+				axe.commons.utils.escapeSelector(element.getAttribute('id')) +
```

Comment:
The line breaks don't make this much easier to read. Perhaps the escapeSelector line should be indented to indicate a hierarchy?